server/src/main/java/org/opensearch/action/search/TransportSearchAction.java

```diff
@@ -493,12 +502,21 @@ private void executeRequest(
 
         final List<SearchRequestOperationsListener> searchListenersList = createSearchListenerList(originalSearchRequest, timeProvider);
         SearchRequestContext searchRequestContext = new SearchRequestContext(
+            originalSearchRequest,
             new SearchRequestOperationsListener.CompositeListener(searchListenersList, logger)
         );
         searchRequestContext.getSearchRequestOperationsListener().onRequestStart(searchRequestContext);
 
         PipelinedRequest searchRequest;
         ActionListener<SearchResponse> listener;
+        Span requestSpan;
+
+        if (FeatureFlags.isEnabled(TELEMETRY)) {
+            requestSpan = tracer.startSpan(SpanBuilder.from("coordinatorRequest"));
+            originalListener = TraceableActionListener.create(originalListener, requestSpan, tracer);
+            searchRequestContext.setRequestSpan(requestSpan);
```

Comment:
Where are you adding this span to a scope?

Reply:
Added scope for `requestSpan`